nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/rdbmodern/dto/NrtObservationDto.java

```diff
@@ -220,4 +220,22 @@ public class NrtObservationDto {
 
     @SerializedName("max_datetime")
     private String maxDatetime;
```

Comment:
maxDatetime is system column. Are there any issues when this is polled?

Reply:
It should be ok and won't cause any problem as it's only in the dto class , not in jpa class. It's commented in the JPA class. I guess this field is not being used, we can cleanup later once DUC is available next week.